app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -0,0 +1,94 @@
+package org.fossasia.openevent.data;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Created by MananWason on 27-05-2015.
+ */
+public class Event {
+    int id;
+    String name;
+    String logo;
+    SimpleDateFormat start;
+    SimpleDateFormat end;
+    float latitude;
+    float longitude;
+    String location_name;
```

Comment:
I am using the Hungarian Notation as far as I remember it. What do you mean by putting @SerializedName("location_name") as an annotation. What purpose would that serve?